ansible/os-devspace.yml

```diff
@@ -0,0 +1,92 @@
+---
+# Playbook for starting a docker instance in the
+# openstack cloud.
+
+## See os-create.yml
+
+## Run:
+# $ source openrc.sh
+# $ ansible-playbook os-devspace.yml -e vm_name=test-devspace -e vm_key_name=yourprofile
+
+- hosts: localhost
+  connection: local
+  #gather_facts: false
+
+  vars:
+
+    vm_image: "CentOS 7 1607"
+    vm_key_name: "jenkins"
+    vm_flavour: m2.large
+    vm_size: 50
+    vm_groups: "ansible-managed,os-image-centos,docker-hosts,devspace"
```

Comment:
I tried to convince myself of that recently, but then could think of uses. Let's go through that when we add the groups README. Some of this information could also be stored in the openstack properties:

```
openstack server set --property ansible_host_vars="ansible_ssh_user->centos;ansible_sudo->yes" josh-devspace
```
```